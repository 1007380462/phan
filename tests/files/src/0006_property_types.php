<?php

class A {

    /**
     * @var bad message that is not a type
     */
    public static $property = [
        'alpha',
        'beta',
        'gamma'
    ];
}
