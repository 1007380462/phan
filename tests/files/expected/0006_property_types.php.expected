%s:6 PhanTypeMismatchProperty Assigning string[] to property but \a::property is \bad
%s:9 PhanUndeclaredTypeProperty Property of undeclared type \bad
